src/handlers/RF24DualLogHandler.cpp

```diff
@@ -55,15 +55,3 @@ void RF24DualLogHandler::setLogLevel(uint8_t logLevel)
    handler1->setLogLevel(logLevel);
    handler2->setLogLevel(logLevel);
 }
-
-void RF24DualLogHandler::write(uint8_t logLevel, const __FlashStringHelper *vendorId,
-         const char *message, va_list *args)
-{
-   // no implementation needed; log messages are redirected to handlers in custom log() method
-}
-
-void RF24DualLogHandler::write(uint8_t logLevel, const __FlashStringHelper *vendorId,
-         const __FlashStringHelper *message, va_list *args)
-{
-   // no implementation needed; log messages are redirected to handlers in custom log() method
-}
```

Comment:
I tried this on my dev docs branch and got an entirely different error (not warning) when compiling it. Mainly because we're still getting `info()`, `debug()`, etc from inheriting the RF24LogHandler in this class.

In my PR to your fork I added a Arduino CI workflow so we don't have to manually compile the sketches for each change, the workflow will tell us if the sketches fail to compile and for what boards.